javascripts/main.js

```diff
@@ -14,9 +14,7 @@ define([
     projectsPanel = null;
 
   var getFilterUrl = function() {
-    return location.href.indexOf('/#/filters') > -1
-      ? location.href
-      : location.href + 'filters';
+    return location.href + (location.href.includes('/#/filters') ? "" + "filters");
```

Comment:
There's something not right here which is why CI isn't currently passing:

```
$ npm run lint

/home/runner/work/up-for-grabs.net/up-for-grabs.net/javascripts/main.js
  17:82  error  Parsing error: Unexpected token )
```

What were we trying to do here?

Reply:
Hi @shiftkey,

I've fixed the typo in my code - see the latest commit.

Jack